extension/war/src/main/webapp/groovy/social/webui/composer/UIComposer.gtmpl

```diff
@@ -36,12 +36,13 @@
     String spaceGroupId = uicomponent.getSpaceGroupId();
     String currentUserId = _ctx.getRequestContext().getRemoteUser();
     isCurrentUserRedactor = SpaceUtils.isRedactor(currentUserId, spaceGroupId);
+    isCurrentUserManager = SpaceUtils.isManager(currentUserId, spaceGroupId);
```

Comment:
use `spaceService.isManager` & `spaceService.isRedactor` instead of redoing an algorithm

Reply:
Added as a remark above